draft-quic-atsss-reqs.mkd

```diff
@@ -309,7 +309,7 @@ Legend:
 
 This approach provides 0-RTT (Zero Round-Trip Time) conversion service since no extra delay is induced by the Convert protocol compared to connections that are not proxied. Also, the Convert Protocol does not require any encapsulation (no tunnels, whatsoever).
 
-# ATSSS Version 2
+# ATSSS Version 2: Handling Non-TCP
```

Comment:
Should it be Handling TCP and non-TCP ?

Reply:
Let's keep it simple and focus on non-TCP case to complement what is already supported

Reply:
Would prefer to mention also 3GPP Rel

e.g. ATSSS Rel. 17: Handling non-TCP

Reply:
I updated the text to introduce 3GPP releases.

Reply:
I feel a unified QUIC solution for TCP and non-TCP flows is envisioned on the mid-term. This impacts the requirements we will be defining later, I would agree to include TCP in that version 2, but maybe this does not correspond to Release 17 yet ?

Reply:
What's about:

Adding non-TCP support

which leaves open for the time being, if we envisage a unified or complementary solution?

Reply:
> 
> 
> What's about:
> 
> Adding non-TCP support
> 
> which leaves open for the time being, if we envisage a unified or complementary solution?

Deal. Updated the text accordingly.